### ISSUE TITLE
JSON formatting?

### ISSUE THREAD

**Issue by Scotchester** [cfpb/front-end#16]
Do we have a stance on JSON formatting? Should it be part of this groups purview to set one?

Because... [yuck](https://github.com/cfpb/pantheon-helpers/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R179) :)

**Comment by jimmynotjim** [comment]
Yeah, that's really hard to read. I've never seen a formatting guide but I've gone by the new line rule for any brace, whether it's curly or square.

**Comment by cjfont** [comment]
JavaScript's built-in `JSON.stringify()` method has the ability to do pretty-printing if you specify an indentation amount: `JSON.stringify(obj, null, 2)`.  Its output is pretty much what @jimmynotjim describes.

**Comment by ascott1** [comment]
Google has a pretty great [JSON style guide](https://google-styleguide.googlecode.com/svn/trunk/jsoncstyleguide.xml). I'd vote for adopting that or forking it and customizing it for these docs. Thoughts?

**Comment by ascott1** [comment]
Let's link to the Google guide in our repo.

**Comment by KimberlyMunoz** [comment]
The link changed: https://google.github.io/styleguide/jsoncstyleguide.xml
